{"timestamp":"2025-07-13T21:52:01.111758","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-13T21:52:01.112526","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/dag_generador_datos.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-13T21:52:01.134779","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"UniqueViolation","exc_value":"duplicate key value violates unique constraint \"service_pkey\"\nDETAIL:  Key (id)=(1) already exists.\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":867,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1159,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/dag_generador_datos.py","lineno":74,"name":"generar_servicios_dispositivos"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-07-13T21:52:01.206244Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T21:52:01.206489Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T21:52:01.206638Z","level":"info","event":"Current task name:generar_servicios","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T21:52:01.206778Z","level":"info","event":"Dag name:cargar_d","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T21:52:01.206913Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T21:52:01.207047Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T21:52:01.207155Z","level":"info","event":"Task:<Task(PythonOperator): generar_servicios>","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T21:52:01.207253Z","level":"info","event":"Failure caused by duplicate key value violates unique constraint \"service_pkey\"","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T21:52:01.207395Z","level":"info","event":"DETAIL:  Key (id)=(1) already exists.","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T21:52:01.207494Z","level":"info","event":"","chan":"stdout","logger":"task"}
